src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -152,7 +166,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             Instant.now());
 
     messageStore.addMessage(message);
```

Comment:
There's weird indentation again, did you say this was due to your editor?

Reply:
Yeah, it keeps doing that because of the way it handles tabs/spaces. I think I'm going to switch editors since it's getting annoying.